cfn.yaml

```diff
@@ -314,19 +314,40 @@ Resources:
       AlarmActions:
         - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:fulfilment-dev
       AlarmName: !Sub High 5XX rate for manage-frontend in ${Stage}
```

Comment:
Please could we improve the `AlarmName` and `AlarmDescription` to match others in Reader Revenue, see https://github.com/guardian/members-data-api/pull/426

Reply:
sure will do

Reply:
done:

<img width="1165" alt="Screenshot 2020-03-18 at 10 39 53" src="https://user-images.githubusercontent.com/10913420/76952541-2a151880-6905-11ea-99e2-9900434c8110.png">

<img width="339" alt="Screenshot 2020-03-18 at 10 40 33" src="https://user-images.githubusercontent.com/10913420/76952550-2ed9cc80-6905-11ea-9427-5714328b4570.png">